File: codewars/Python/7kyu/Alternate capitalization.py
# ALTERNATIVE SOLUTION
# def capitalize(s):
#     s = ''.join(c if i%2 else c.upper() for i,c in enumerate(s))
#     return [s, s.swapcase()]

def capitalize(s):
    even = ''
    odd = ''
    
    for i, ch in enumerate(s):
        if i % 2 == 0 and i != 1:
            even += ch.upper()
            odd += ch
        else:
            even += ch
            odd += ch.upper()
            
    return [even, odd]

File: codewars/JavaScript/7kyu/Alternate capitalization.js
function capitalize(s){
    let odd = ''
    let even = ''
    
    for(let i = 0; i < s.length; i++) {
      if(i % 2 === 0 && i !== 1) {
        even += s[i].toUpperCase()
        odd += s[i]
      } else {
        even += s[i]
        odd += s[i].toUpperCase()
      }
    }
    
    return [even, odd]
  }